src/GameFace/Models/Achievements.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Threading.Tasks;
+
+namespace GameFace.Models
+{
+    public class Achieve
+    {
+        [DataMember]
+        public int idAchieve { get; set; }
+        [DataMember]
+        public int idTask { get; set; }
+        [DataMember]
+        public int levelNeeded { get; set; }
+        [DataMember]
+        public string description { get; set; }
+        [JsonIgnore] 
+        [IgnoreDataMember]
+        public virtual Tasks tasks { get; set; }
+        [JsonIgnore]
+        [IgnoreDataMember]
```

Comment:
I'm also not sure if you need the IgnoreDataMember attribute - if you do, then we will probably need them for the other as well